modules/tinymce/src/plugins/help/main/ts/ui/PluginsTab.ts

```diff
@@ -55,14 +50,15 @@ const tab = (editor: Editor): Dialog.TabSpec => {
   const makeLink = (p: {name: string; url: string}): string =>
     `<a href="${p.url}" target="_blank" rel="noopener">${p.name}</a>`;
 
-  const maybeUrlize = (editor: Editor, key: string) => Arr.find(PluginUrls.urls, function (x: PluginUrlType) {
+  const maybeUrlize = (editor: Editor, key: string) => Arr.find(PluginUrls.urls, function (x: PluginUrls.PluginUrlType) {
     return x.key === key;
   }).fold(function () {
     const getMetadata = editor.plugins[key].getMetadata;
     return typeof getMetadata === 'function' ? makeLink(getMetadata()) : key;
   }, function (x) {
     const urlSlug = x.slug || x.key;
-    return makeLink({ name: x.name, url: 'https://www.tiny.cloud/docs/plugins/' + urlSlug });
+    const type = x.type || 'opensource';
+    return makeLink({ name: x.name, url: 'https://www.tiny.cloud/docs/plugins/' + type + '/' + urlSlug });
```

Comment:
Can we do a template literal here?

```suggestion
    return makeLink({ name: x.name, url: `https://www.tiny.cloud/docs/plugins/${type}/${urlSlug}` });
```